### PR TITLE
Woo! German docs! 🙌

### PR DESCRIPTION
**Pull request by matchai**

### CODE REVIEW

File: tweets/2020/01/german-docs.tweet [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Starship's docs are now translated in German! 🇩🇪🎉
+
```
Comment (jletey) [inline tweets/2020/01/german-docs.tweet]:
```suggestion
```

In the Russian tweet, we didn't have a new line - if you're looking for consistency

Comment (matchai) [reply tweets/2020/01/german-docs.tweet]: I made this PR in the GitHub UI, which seems to auto-insert this line-break.
It looks like it doesn't show up in the preview: https://github.com/starship/twitter/pull/4/checks?check_run_id=407347834

Comment (jletey) [reply tweets/2020/01/german-docs.tweet]: The link from above shows a newline for me, v.s. the preview of the Russian tweet: https://github.com/starship/twitter/pull/3/checks

Comment (matchai) [reply tweets/2020/01/german-docs.tweet]: Oh, _that_ newline. I thought you meant the trailing newline. 😅 
Good call 👍

Comment (jletey) [reply tweets/2020/01/german-docs.tweet]: Sorry for the miss-understanding 😅

**Review by jletey — APPROVED** [review]
LGTM 👍

**Pull request merged**